### ISSUE TITLE
Not sniping from PokeSnipers

### ISSUE THREAD

**Issue by s10035147** [5andr0/PogoLocationFeeder#140]
![image](https://cloud.githubusercontent.com/assets/20797028/17350900/fe17946c-595c-11e6-8ddb-2bc395bc68c0.png)

Its only sniping from discord channels and not from PokeSnipers. Or am I missing something?

**Comment by djdookie** [comment]
I think cloudflare is preventing the bot's requests to pokesnipers.com now.

**Comment by Ilses** [comment]
Yep -- the change to Cloudflare broke our current integration. I'll close this when we've looked into it.
